src/components/MessageBox/MessageBox.spec.js

```diff
@@ -15,3 +22,26 @@ describe('MessageBox component', () => {
 		expect(okBTN).toBeInTheDocument();
 	});
 });
+
+describe('MessageBox component with type "start"', () => {
+	it('should render children', () => {
+		render(<MessageBox type="start" text="hey" />);
+		const msg = screen.getByText('hey');
+		const cancelBTN = screen.getByText('cancel');
+		const okBTN = screen.getByText('ok');
+
+		expect(msg).toBeInTheDocument();
+		expect(cancelBTN).toBeInTheDocument();
+		expect(okBTN).toBeInTheDocument();
+	});
+});
+
+describe('Button at the MessageBox component with type "start"', () => {
+	it('should allow clicks', async () => {
+		render(<MessageBox type="start" text="hey" />);
+		const okBTN = screen.getByText('ok');
+
+		await userEvent.click(okBTN);
+		expect(mockedUsedNavigate).toHaveBeenCalledTimes(1);
+	});
+});
```

Comment:
Since you don't call a callback funtion, you don't necessarily need to test that the button is clickable. I think 🤔